README.md

```diff
@@ -49,23 +46,41 @@ async def main():
                     "OUTPUT_DIR": "/golem/output",
                 },
             )
-            ctx.run("/golem/entrypoints/render_entrypoint.py")
-            ctx.download_file("/golem/output/out.png", f"output_{frame}.png")
-            yield ctx.commit(task)
-            # TODO: Check if job is valid
-            # and reject by: task.reject_task(reason = 'invalid file')
-            task.accept_task()
+            ctx.run("/golem/entrypoints/run-blender.sh")
+            output_file = f"output_{frame}.png"
+            ctx.download_file(f"/golem/output/out{frame:04d}.png", output_file)
+            yield ctx.commit()
+            task.accept_task(result=output_file)
+
+        ctx.log("no more frames to render")
 
-        ctx.log("no more frame to render")
+    # iterator over the frame indices that we want to render
+    frames: range = range(0, 60, 10)
+    init_overhead: timedelta = timedelta(minutes=3)
 
+    # By passing `event_emitter=log_summary()` we enable summary logging.
+    # See the documentation of the `yapapi.log` module on how to set
+    # the level of detail and format of the logged information.
     async with Engine(
         package=package,
-        max_worker=10,
+        max_workers=3,
         budget=10.0,
-        timeout=timedelta(minutes=5),
+        timeout=init_overhead + timedelta(minutes=len(frames) * 2),
```

Comment:
This way it's easier to understand that the `timeout` parameter is the timeout for the whole computation. :)

Reply:
well, I only moved code from the actual example